src/test/java/edu/hawaii/its/api/service/TestUpdateMemberService.java

```diff
@@ -442,6 +442,16 @@ public void checkIfSelfOptOrAdminTest() {
 
     }
 
+//    @Test
+//    public void addRemovePathOwnershipsTest(){
+//        // Todo integration test for updating grouping owners with path owners
+//    }
+
+//    @Test
+//    public void checkAllMembersAfterAddRemovePathOwnershipsTest(){
+//        // Todo integration test for checking the changes of members' group after adding and removing path owners
+//    }
```

Comment:
If you are planning to add the integration tests in another ticket, could you at the very least add tests for the wrapper classes (`AddMembersCommand.java` amd `RemoveMembersCommand.java`) and an integration test inside `TestGroupingsRestControllerv2_1.java`? Thanks.

Reply:
I just added just TODO tests inside of the TestGroupingsRestControllerv2_1.java.